### ISSUE TITLE
Pol mode xcorr pol cal for flexure should consider larger flexure shifts

### ISSUE THREAD

**Issue by dsavransky** [geminiplanetimager/gpi_pipeline#99]
We encountered on the last llp run that the pol cal taken the day before needed a [dx, dy] ~= [-2.5,1.5] shift in flexure to get centered on the pol spots. This perhaps indicates we should increase the range of possible flexure shifts the xcorr routine considers in pol mode. Since 3 pixels is ~lambda/D on the raw image, and since the pol mode spots are pretty well spaced apart, 3 pixel offsets shouldn't cause the xcorr to find an incorrect set of spots?

**Comment by mperrin** [comment]
Legacy issue migrated from redmine issue tracker. Originally filed by @semaphoreP on 2016-04-01

**Comment by semaphoreP** [comment]
@maxwellmb, what is your availability to work on this? I recall this was a issue in some runs, but not others. I would like to have this fixed, but it's not a blocking issue, so I was wondering when do you see yourself working on this?

**Comment by maxwellmb** [comment]
@astrofitz didn't you have a new student you was maybe going to look into this?

**Comment by arriagap** [comment]
Yes, Kevin Stahl should be working on it, but I don't think he's been added here. I can't tag him. HIs git name is kstahl93
